docs/release.md

```diff
@@ -0,0 +1,174 @@
+    production, a data migration will be required and must be tested before
+    running in production.
+    - CI pipeline successfully completes.
+5. Developer makes Pull Request for the team to review
+6. Team reviews and approves the Pull Request
+7. Developer merges feature branch into `master` branch
+
+### Testing
+
+The steps in the [Pre-Release](#Pre-Release) include a rigorous testing process
+which includes unit tests and integration tests (read further for details) so
+that there is a very low chance of failure or issues with the FHIR model
+resources themselves when the FHIR model is deployed to the production FHIR
+server.
+
+This helps ensure that clients and users of the FHIR services in production
+will have a high quality experience with minimum bugs in the FHIR model.
+
+## Release Process
+
+## Executed by `fhirmodel release` command
+
+A new CLI command will be added to facilitate the release of the FHIR model
+
+```shell
+# Command spec
+$ fhirmodel release <IG control file> <version type>
+
+# Example
+$ fhirmodel release site_root/ig.ini minor
+```
+
+When this command is run, the following steps will be executed:
+
+1. **Generate the next release version number**
+
+    - CLI will use the last release's version number and the version type
+    to generate the next version number
+
+2. **Prep conformance resources for release**
+
+    - The `version` attribute on all conformance resources will be populated
+      with the new version number.
+    - The `publisher` attribute on all conformance resources with the
+      Kids First DRC canonical URL will be populated with value in
+      `ImplementationGuide.publisher`
+    - The `contact` attribute on all conformance resources with the
+      Kids First DRC canonical URL will be populated with appropriate information
+    - The `date` attribute on all conformance resources will be populated with
+      the current date.
+
+3. **Produce the NPM package for the model**
+
+    - Run the IG publisher to produce the NPM package for the model
+    - Create a tarball of the NPM package
+
+4. **Update `CHANGELOG.md` with release notes**
+
+    - Generate release notes using titles of Git Pull Requests merged into
+      `master` branch since last release
+    - See [current Kids First release maker](https://github.com/kids-first/kf-release-maker/tree/refactor-release-maker) for an example.
+
+5. **Create a release-version branch and Pull Request**
+
+    - Create a release-version branch off of `master`: `release-<version number>`
+    - Create a Pull Request to merge this branch into `release` branch
+    - Set the PR description to the release notes markdown
+
+## Executed by FHIR modeling team
+
+6. **Team must review and approve the Pull Request**
+
+    - The only FHIR model resource changes to review at this point will be
+      those made in [Release Process step 2](#Release-Process).
+
+    - The team must also review the release note changes in `CHANGELOG.md`.
```

Comment:
Why do these need to be reviewed?

Reply:
The same reason we review them now in our releases. Sometimes we look at them and see that something is missing and we want to wait for one more PR to merge before cutting a release.